lib/src/html_generator.dart

```diff
@@ -523,6 +528,13 @@ String renderMarkdown(String markdown, {nestedContext}) {
   return doc.body.innerHtml;
 }
 
+/// Convert the given plain text into HTML.
+String renderPlainText(String text) {
+  if (text == null) return '';
+
+  return "<code class='fixed'>${text.trim()}</code>";
```

Comment:
And to clarify, if we were to use `pre` here, some selectors would trigger which would give it a different background than normal doc text as well as a border. It would look like inset code rather than documentation.

Conversely, `code.fixed` looks like documentation (rather than in-line code). So it renders plain text files (readme.txt, license.txt, ...) better. I would recommend sticking with the current `code.fixed` style, but will change it to whatever is desired.